src/components/activity/ActivityDecomposition.svelte

```diff
@@ -1,50 +1,75 @@
 <svelte:options immutable={true} />
 
 <script lang="ts">
+  import CaretDownIcon from '@nasa-jpl/stellar/icons/svg/caret_down.svg?component';
```

Comment:
Can we add a unit test for this file?

Reply:
Yeah was wondering about the best way to do that. Don't see any "testing-library" tests with a mock store. Might be possible but maybe for now I just use the other testing library?

Reply:
Which one?

Reply:
Oh scratch that I was looking at the e2e tests. I'll look into mocking the store which is needed by the ActivityDecomposition component.

Reply:
Yeah actually an e2e test would probably be even more useful! I'm good with either as long as we have some base signal for correctness

Reply:
Added a component level test by adding to activities to the activity store before the component test.